src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -1649,6 +1649,28 @@ public static function provideRenderVerticalTests(): \Traversable
             $books,
         ];
 
+        yield 'With multibyte characters in some headers (the "í" in "Dívíne")' => [
```

Comment:
This is not reproducing the bug as `Dívíne` is in the value, not in the header.

Reply:
But the `í` in `Títle` is in the header, isn't it?

Reply:
Ah yes, you are right. I’ll update the fixture name to „Títle“ as Alexander figured correctly.

Reply:
should finally be fixed now